### PR TITLE
update mapper framework config file

### PR DESCRIPTION
**Pull request by wbc6080**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Currently, mapper-framework configures mapper parameters through the config.yaml file and put the config file into the mapper image. Users cannot change the configuration information unless rebuilding the image. This pr mounts the config configuration information into the mapper pod in the form of configmap, which can easily change the mapper configuration method.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: staging/src/github.com/kubeedge/mapper-framework/_template/mapper/resource/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    grpc_server:
+      socket_path: /etc/kubeedge/Template.sock
+    common:
+      name: Template-mapper
+      version: v1.13.0
+      api_version: v1.0.0
+      protocol: # TODO add your protocol name
+      address: 127.0.0.1
+      edgecore_sock: /etc/kubeedge/dmi.sock
+    dev_init:
```
Comment (cl2017) [inline staging/src/github.com/kubeedge/mapper-framework/_template/mapper/resource/configmap.yaml:16]: now we can remove this field `dev_init`

Comment (wbc6080) [reply staging/src/github.com/kubeedge/mapper-framework/_template/mapper/resource/configmap.yaml:16]: OK I will fix it

**Comment by wbc6080** [comment]
PTAL @cl2017 @WillardHu

**Comment by WillardHu** [comment]
Please mke the unit test to successful and you can close-reopen this pr to retry pipeline checks.

**Pull request merged**